iso/Dockerfile

```diff
@@ -45,8 +45,15 @@ RUN for dep in $HOST_FOLDER_DEPS; do \
         rm -f "/tmp/$dep.tcz"; \
     done
 
+# Add Hyper-V Data Exchange client
+RUN curl -fSL -o "/tmp/go-hvkvp.tcz" "https://files.gbraad.nl/hvkvp/go-hvkvp-0.0.3.tcz"; \
```

Comment:
Wondering if we find a place for this where other contributors can get access if any modification required. This is fine for testing and long term we need place for this.